### PR TITLE
KomaMRI Proposal for JSoC

### PR DESCRIPTION
**Pull request by beorostica**
We are thrilled to participate in JSoC/GSoC. This is our proposal.

### CODE REVIEW

File: jsoc/gsoc/juliahealth.md [hunk 1]
```diff
@@ -83,3 +83,39 @@ Furthermore, although not required, publishing in the JuliaCon Proceedings will
 Additionally, depending on the success of the package, there is a potential to run experiments on actual patient data to generate actual patient population insights based on a chosen research question. 
 This could possibly turn into a separate research paper, conference submission, or poster submission. 
 Whatever may occur in this situation will be supported by project mentors. 
+
+### Project 3: Highly-efficient MRI Simulations with Multi-Vendor GPU Support
+
+**Description:** 
+KomaMRI.jl is a Julia package designed for highly-efficient Magnetic Resonance Imaging (MRI) simulations, serving both educational and research purposes.
+
+Simulations can help to grasp hard-to-understand MRI concepts, like signal acquisition, and image reconstruction. Moreover, they can guide the design of novel pulse sequences, and generate synthetic data for training machine learning models.
```
Comment (cncastillo) [inline jsoc/gsoc/juliahealth.md:92]:
```suggestion
KomaMRI.jl is a Julia package designed for highly-efficient Magnetic Resonance Imaging (MRI) simulations, serving both educational and research purposes. Simulations can help to grasp hard-to-understand MRI concepts, like signal acquisition, and image reconstruction. Moreover, they can guide the design of novel pulse sequences, and generate synthetic data for training machine learning models.
```
Thanks @beorostica for the PR, I think these can be in the same paragraph.

**Comment by TheCedarPrince** [comment]
Could you give me review permissions here @ViralBShah ? I was going to make a review but realized I couldn't be requested as a reviewer. Thanks!

**Comment by ViralBShah** [comment]
Invited.

**Comment by TheCedarPrince** [comment]
Awesome -- I'll take a look in a couple hours. Thanks @ViralBShah!

**Comment by giordano** [comment]
> I was going to make a review but realized I couldn't be requested as a reviewer. Thanks!

For what is worth, you don't need to be a collaborator of a repository to do a review.  @cncastillo did a review above for example.

File: jsoc/gsoc/juliahealth.md [hunk 2]
```diff
@@ -83,3 +83,37 @@ Furthermore, although not required, publishing in the JuliaCon Proceedings will
...
+KomaMRI.jl is a Julia package designed for highly-efficient Magnetic Resonance Imaging (MRI) simulations, serving both educational and research purposes. Simulations can help to grasp hard-to-understand MRI concepts, like pulse sequences, signal generation and acquisition. Moreover, they can guide the design of novel pulse sequences, and generate synthetic data for training machine learning models.
+
+Currently, our simulator performs GPU-accelerated computations using CUDA arrays. We are now advancing to implement a new simulation method (`BlochKernel<:SimulationMethod`) based on GPU kernel programming using KernelAbstractions.jl. This enhancement will not only boost computation speeds but also broaden accessibility with KernelAbstractions.jl's multi-vendor GPU support. This could enable the use of MRI simulations in iterative algorithms to solve inverse problems. We are seeking enthusiastic people interested in developing this functionality.
+
+- **Mentors:** Carlos Castillo [email: cncastillo@uc.cl], Boris Oróstica [email: beorostica@uc.cl], Pablo Irarrazaval [email: pim@uc.cl]
+
+- **Difficulty:** Hard
+
+- **Duration:** 320 hours (2 months, 8 hours per day)
```
Comment (TheCedarPrince) [inline jsoc/gsoc/juliahealth.md:98]: Could you change this to 350 hours?

**Review by TheCedarPrince — CHANGES_REQUESTED** [review]
Once you address my small comment and also fix the merge conflict (this should go under the "Medical Imaging Subecosystem Projects" section) this will be ready to merge. Thanks @beorostica

File: jsoc/gsoc/juliahealth.md (hunk 2 above)
Comment (beorostica) [reply jsoc/gsoc/juliahealth.md]: Addressed in a899540 and  7a8c199

**Pull request merged**